docs/shared/reference/nx-json.md

```diff
@@ -162,6 +163,8 @@ defining `targetDefaults` in `nx.json` is helpful.
 
 The configuration above is identical to adding `{"dependsOn": ["^build"]}` to every build target of every project.
 
+For full documentation of the `dependsOn` property, see the [project configuration reference](/reference/project-configuration#dependson).
```

Comment:
Do you think we can just inline that documentation here? I'm afraid folks might get confused